packages/backend/src/peripherals/aztec/utils.ts

```diff
@@ -0,0 +1,9 @@
+import { assert } from '@l2beat/common'
+
+import { Block, Rollup, Rollups, toBlock } from './schemas'
+
+export function findLatestBlock(rollups: Rollups): Block {
+  const latestMined = rollups.find((r): r is Rollup => r.mined !== null)
+  assert(latestMined, 'No mined block found in first 10 blocks')
```

Comment:
It isn't obvious why this error occurs. Maybe the name of the function has to change? Also, if the blocks aren't mined shouldn't we just query for more?

Reply:
> It isn't obvious why this error occurs. Maybe the name of the function has to change?
`findLatestBlockOrThrow` would work?